app/assets/stylesheets/application/modules/_projects.scss

```diff
@@ -0,0 +1,4 @@
+.c-filters {
+  // Override the filter card's usual large bottom margin, so it looks proportional when rendered as part of the Project view
+  margin-bottom: 1em;
+}
```

Comment:
Pretty sure this will override that everywhere, even though it's in the `projects` module.  (We load all CSS on all pages pretty much.)  Probably better to just add `.mb-1` to the filter div on the projects page to achieve this in one place.